video/out/aspect.c

```diff
@@ -31,8 +31,6 @@ static void aspect_calc_panscan(struct mp_vo_opts *opts,
                                 int window_w, int window_h, double monitor_par,
                                 int *out_w, int *out_h)
 {
-    w *= monitor_par;
```

Comment:
This is static function, you could remove also unused parameter.

Reply:
Passing just the height here where all other params expect w+h pair is a bit weird. And it'll be less work/annoyance if `w` becomes actually used again.

Reply:
Yes, I agree. I don't know why w is not used though. But I don't want to dig into it.